endpointmanager/pkg/archivefile/archivefile.go

```diff
@@ -16,21 +17,22 @@ import (
 
 // totalSummary is the format of a given URL's JSON object for the archive file
 type totalSummary struct {
-	URL                string                 `json:"url"`
-	CreatedAt          time.Time              `json:"created_at"`
-	ListSource         []string               `json:"list_source"`
-	OrganizationNames  []string               `json:"api_information_source_name"`
-	Updated            map[string]interface{} `json:"updated_at"`
-	NumberOfUpdates    int                    `json:"number_of_updates"`
-	Operation          map[string]interface{} `json:"operation"`
-	FHIRVersion        map[string]interface{} `json:"fhir_version"`
-	TLSVersion         map[string]interface{} `json:"tls_version"`
-	MIMETypes          firstLastStrArr        `json:"mime_types"`
-	Vendor             map[string]interface{} `json:"certified_api_developer_name"`
-	ResponseTimeSecond interface{}            `json:"median_response_time"`
-	HTTPResponse       []httpResponse         `json:"http_response"`
-	SmartHTTPResponse  []smartHTTPResponse    `json:"smart_http_response"`
-	Errors             []responseErrors       `json:"errors"`
+	URL                  string                 `json:"url"`
+	CreatedAt            time.Time              `json:"created_at"`
+	ListSource           []string               `json:"list_source"`
+	OrganizationNames    []string               `json:"api_information_source_name"`
+	Updated              map[string]interface{} `json:"updated_at"`
+	NumberOfUpdates      int                    `json:"number_of_updates"`
+	Operation            map[string]interface{} `json:"operation"`
+	FHIRVersion          map[string]interface{} `json:"fhir_version"`
+	TLSVersion           map[string]interface{} `json:"tls_version"`
+	MIMETypes            firstLastStrArr        `json:"mime_types"`
+	Vendor               map[string]interface{} `json:"certified_api_developer_name"`
+	ResponseTimeSecond   interface{}            `json:"median_response_time"`
+	HTTPResponse         []httpResponse         `json:"http_response"`
+	SmartHTTPResponse    []smartHTTPResponse    `json:"smart_http_response"`
+	Errors               []responseErrors       `json:"errors"`
+	RequestedFhirVersion string                 `json:"requested_fhir_version"`
```

Comment:
This is incredibly nitpicky, but I think it makes sense to move `RequestedFhirVersion` to right below `URL`, since this is the order the fields in the JSON file are written in, and I think the url and requested fhir version should be at the top since that's what makes each object unique.